src/marine/glider2ioda.py

```diff
@@ -20,33 +20,32 @@
     IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
 sys.path.append(str(IODA_CONV_PATH.resolve()))
 
-import ioda_conv_ncio as iconv
+import ioda_conv_engines as iconv
 from orddicts import DefaultOrderedDict
 
 vName = [
     "sea_water_temperature",
     "sea_water_salinity"]
 
-
 locationKeyList = [
     ("latitude", "float"),
     ("longitude", "float"),
     ("depth", "float"),
     ("datetime", "string")
 ]
 
-AttrData = {
+GlobalAttrs = {
     'odb_version': 1,
 }
 
 
 class Profile(object):
 
-    def __init__(self, filename, date, writer):
+    def __init__(self, filename, thin, date):
```

Comment:
What does the `writer` argument do and why did it have to be removed?

Reply:
**the writer argument was before upgrading the converter to V2:**
import ioda_conv_ncio as iconv
# setup the IODA writer
writer = iconv.NcWriter(args.output, locationKeyList)
# Read in the input data
my_class = MyClass(args.input, writer)

**after upgrading, ioda-engine is used , they ought to change accordingly that was to replace writer argument with:**
import ioda_conv_engines as iconv
# Read in the input data
my_class = MyClass(args.input)

So that is why writer was removed.

Reply:
Thank you.